src/objects.js

```diff
@@ -1,39 +1,72 @@
 export const createPerson = (name, age) => {
-  // your code here
+  // creates an object with the given name and age properties
+  let person = {
+    name: name,
+    age: age
+  };
+
+  return (person);
+
 };
 
 export const getName = (object) => {
-  // your code here
+  // returns the name property of the object
+  return(object["name"]);
+
 };
 
 export const getProperty = (property, object) => {
-  // your code here
+  return(object[property]);
```

Comment:
💯This is a great example of where square bracket notation would be preferred to dot notation.